core/templates/dev/head/pages/collection-editor-page/collection-editor-page.mainpage.html

```diff
@@ -1,37 +1,33 @@
-{% extends 'dist/base.html' %}
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    @require('../../base_components/header.html', {"title": "Oppia"})
```

Comment:
does this relative path work in prod too? just confirming.

Reply:
Yes, this is only used in the build process. We will hopefully soon change this to absolute imports.